### ISSUE TITLE
Update jwt-go depdenceny

### ISSUE THREAD

**Issue by arekkas** [ory-am/fosite#51]
```
src/github.com/ory-am/fosite/token/jwt/jwt.go:29: cannot use claims.ToMap() (type map[string]interface {}) as type jwt.Claims in assignment:
map[string]interface {} does not implement jwt.Claims (missing Valid method)
```

**Comment by plandem** [comment]
same issue on go 1.6.1

**Comment by arekkas** [comment]
thank you for the report @plandem . Please use the :+1: feature if this issue affects you in the future. The issue is not related to the go version but due to breaking changes in `jwt-go`. You can use `glide install` to install the right dependencies.

**Comment by plandem** [comment]
sorry, i'm still quite new to Go. i choose https://github.com/kovetskiy/manul as dep manager

**Comment by arekkas** [comment]
No worries, this repository uses glide for dependency management: https://github.com/Masterminds/glide

**Comment by plandem** [comment]
i still can't understand how to fix issue. i'm trying to compile example and just no luck

**Comment by arekkas** [comment]
Thanks for the report, there indeed seems to be something broken with running the example. Additionally, the docs are missing glide. Sorry for that.

Patch #52 resolves the internal import in the example. Now do the follwoing to run the example with glide:

```
go get github.com/Masterminds/glide
go get github.com/ory-am/fosite
# unix
cd $GOPATH/src/github.com/ory-am/fosite
# windows
cd %GOPATH%\src\github.com\ory-am\fosite
glide install
go run fosite-example/main.go
```

Thank you for your patience and reports :) I hope everything works now. If not, let me know.

**Comment by plandem** [comment]
worked finally. thanks.

p.s.: it would be nice to see some hints at read me for 2 step auth process :)

**Comment by arekkas** [comment]
yeah aboslutely. the one thing this repository is missing is good docs. it will be done, promised :)

**Comment by arekkas** [comment]
resolved in #53
